### ISSUE TITLE
Tautological If-statement in PlayByPlay.ipynb

### ISSUE THREAD

**Issue by eswan18** [swar/nba_api#42]
In PlayByPlay.ipynb (in the docs/examples directory) I believe this line:
```python
description = play['HOMEDESCRIPTION'] if not None else play['VISITORDESCRIPTION']
```
will always return `play['HOMEDESCRIPTION']` because `not None` is always true. I suspect it should be
```python
description = play['HOMEDESCRIPTION'] if play['HOMEDESCRIPTION'] is not None else play['VISITORDESCRIPTION']
```
It's possible I'm mistaken, but the current logic doesn't seem right.

@rsforbes

**Comment by swar** [comment]
I will give Randy some time to address this ticket before I look into this ticket, but I agree with Ethan on the change that needs to be implemented in order to fix the bug.

**Comment by jaypeedevlin** [comment]
If you want to be succinct, you can use the `or` operator.  The code below is equivalent to @eswan18's suggestion:

```python
description = play['HOMEDESCRIPTION'] or play['VISITORDESCRIPTION']
```

**Comment by eswan18** [comment]
Yes, that is a nifty way to do this. But personally I think this kind of brevity-at-the-expense-of-clarity leads to bugs (it made the original one harder to notice, certainly). It might be wiser to follow the Zen and be more explicit.

**Comment by eswan18** [comment]
Closed in 5ae1d7a1f
